packages/mgt-chat/src/utils/displayDates.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+/**
+ * Renders a date object in the format YYYY-MM-DD HH:MM AM. Modified from https://github.com/Azure/communication-ui-library/blob/main/packages/storybook/stories/ChatComposite/snippets/Utils.tsx#L128
+ * @param messageDate is the date object.
+ * @returns date string
+ */
+export const onDisplayDateTimeString = (messageDate: Date): string => {
```

Comment:
Perhaps we should use the date formatting utils we have in mgt-components and get some test coverage on those functions?

Reply:
Absolutely! Are we settled on a format for the dates? The utils in mgt-component are mainly _human-readable_ formats like "Today", "Yesterday" etc. This component defaults to YYYY-MM-DD HH:MM AM. I prefer the latter because of the precision. I can move this util to the mgt-components and expand on the unit tests. What do you think?

Reply:
![image](https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/8081536/8624ea1c-2cd2-492c-a735-2398e06a7c94)